TTbarResLookUpTables.py

```diff
@@ -102,28 +102,25 @@ def multi_dict(K, type): # definition from https://www.geeksforgeeks.org/python-
 luts = multi_dict(2, str) #Annoying, but necessary definition of the dictionary
 
 def LoadDataLUTS(bdiscDirectory, Year, VFP, RemoveContam, ListOfLetters, uploadDir):
+    
     contam = ''
     if RemoveContam == True:
         contam = '_ttContaminationRemoved'
-    if Year != 0:
-        for letter in ListOfLetters:
-            for icat in list_of_cats:
-                df = pd.read_csv(uploadDir+'/LookupTables/' + bdiscDirectory + 'mistag_UL' + str(Year-2000) + VFP + '_JetHT' + '_Data'+contam+'_' + icat[:4] + 'inc.csv')
```

Comment:
should the file include the era? for example:

```
uploadDir+'/LookupTables/' + bdiscDirectory + 'mistag_UL' + str(Year-2000) + VFP + '_JetHT' + '_Data'+contam+'_' + icat[:4] + 'inc.csv'
```
 to
```
uploadDir+'/LookupTables/' + bdiscDirectory + 'mistag_UL' + str(Year-2000) + VFP + '_JetHT' + letter + '_Data'+contam+'_' + icat[:4] + 'inc.csv'

Reply:
The LUTs that we'll use will not be based on the individual era to avoid applying mistag rates made from few statistics (for example, 2016 postVFP era F).  This section is fine :)